pandas/tests/frame/test_query_eval.py

```diff
@@ -1051,8 +1051,21 @@ def df(self):
                 "A": [1, 2, 3],
                 "B B": [3, 2, 1],
                 "C C": [4, 5, 6],
+                "C  C": [7, 4, 3],
```

Comment:
can you add a doc-string to this fixture, saying these are all valid backtick quoted strings

Reply:
OK